gulp/tasks/jscs.js

```diff
@@ -0,0 +1,21 @@
+(function() {
+    'use strict';
+
+    var jshint = require('gulp-jshint'),
```

Comment:
You're always welcome, @benpatterson! Nice catch. I did this too quickly and forgot to clean this up. It was intended to all be named for JSRC.

I've squashed and pushed up a fix.